package.json

```diff
@@ -4,13 +4,10 @@
   "description": "Get back longest and current streak on Github prifiles",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "eslint .; exit 0",
-    "install": "npm i -g babel-cli watch jpm",
-    "clean": "find build -regextype posix-egrep -regex \".*\\.(zip|xpi|nex)$\" -type f -delete && rmdir build",
-    "build:js": "babel ./src/inject.js --presets es2015 | tee ./firefox/data/inject.js > ./chrome/src/inject/inject.js ",
-    "build:css": "echo ./firefox/data/ ./chrome/src/inject/ | xargs -n 1 cp ./src/inject.css",
-    "build": "npm run build:js && npm run build:css",
-    "build:watch": "watch \"npm run build\" ./src/",
+    "lint": "eslint .",
+    "prebuild": "npm run lint && rimraf chrome/src/ firefox/data/ build",
+    "build": "webpack -p",
```

Comment:
Here Webpack compresses everything. When submitting a file to Opera and Mozilla store they must be able to read it or you can give them the instructions to get the same result with the compressed code.
I realized that Opera does not even try to build it and do not approves your extension. (I tried with vs1.5 and no response for a week)